File: java/src/main/java/com/health/probes/Controllers/HomeController.java
package com.health.probes.Controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HomeController {
    String message = "container-apps-health-probe-examples-java";

    @GetMapping("/")
    public String index() {
        return message;
    }
}

File: go/controllers/homeController.go
package controllers

import (
    "github.com/gofiber/fiber/v2"
)

type HomeResponse struct {
	Msg string
  }

func HomeController(c *fiber.Ctx) error {
	res := HomeResponse{
		Msg: "container-apps-health-probe-examples-go",
	}
	return c.JSON(res)
}